Sources/PaymentButtons/PaymentButtonFont.swift

```diff
@@ -12,6 +12,10 @@ extension UIFont {
         var errorRef: Unmanaged<CFError>?
         let frameworkBundle = Bundle(for: PaymentButton.self)
 
+        guard UIFont(name: name, size: 10.0) == nil else {
+            return
+        }
+
```

Comment:
I was thinking if it would make sense to call it from the AppDelegate, so the registration logic is available globally throughout the app. This way i think we wouldn’t need to register it again in individual views or components.